lib/puma/cluster/worker.rb

```diff
@@ -115,6 +115,11 @@ def run
 
         while restart_server.pop
           server_thread = server.run
+
+          if @log_writer.debug? && index == 0
+            debug_loaded_extensions "────────────────────────────────── Loaded Extensions - worker 0:"
```

Comment:
I'm not a huge fan of having this long formatted string right in the code

Maybe:

```ruby
debug_loaded_extensions("worker 0")
```

...and let `debug_loaded_extensions` add the `---` padding and such.

Reply:
See [Fix !DRY code](https://github.com/puma/puma/pull/3036/commits/8d2da80a64587dbcf6f3d78a9ee309b974bd16e5)

Not my best code...  Sory for the double push, first was the commit, second was "maybe it's time to rebase" force push

Happy New Year!